app/views/docs/service.phtml

```diff
@@ -32,8 +37,12 @@ $codes          = [
     ],
 ];
 
-$markdown = new \Parsedown();
-$markdown->setSafeMode(true);
+$environment = new Environment(['html_input' => 'escape']);
```

Comment:
Lets initialize the markdown parser as a resource like we do for other reusable resources, and use dependency injection to get it from the controllers, and pass it to the templates as a variable.

We should make sure we initialize a good chunk of extensions when registering the resource. Happy to go over the list together.